src/components/shared/chat.tsx

```
@@ -276,6 +276,31 @@ export function CardsChat({
+  const [, setTick] = React.useState(0);
+
+  const isFormatActive = (command: string) => {
+  return typeof document !== 'undefined'
+    ? document.queryCommandState(command)
+    : false;
+  };
+
+  useEffect(() => {
+  const onSelectionChange = () => {
+  //Bold, italics and underline button should be highlighted when selected in chat
+  const [, setTick] = React.useState(0);
```

Biome (2.3.13)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)